frontend/src/container/Tracim.jsx

```diff
@@ -127,7 +131,10 @@ class Tracim extends React.Component {
     const { props } = this
 
     const fetchGetConfig = await props.dispatch(getConfig())
-    if (fetchGetConfig.status === 200) props.dispatch(setConfig(fetchGetConfig.json))
+    if (fetchGetConfig.status === 200) {
+      props.dispatch(setConfig(fetchGetConfig.json))
+      document.title = fetchGetConfig.json.instance_name
```

Comment:
you are bypassing the custom event process here. Even if it is in the same file as where you handle the custom event, keep using it's process.
It's only a matter of consistency